### PR TITLE
Test fix

### PR DESCRIPTION
**Pull request by bbelderbos**
check the commit messages for more info ...

### CODE REVIEW

File: tests/test_read.py [hunk 1]
```diff
@@ -3,18 +3,18 @@
...
-    file = os.path.join('.', 'payloads', 'tremor_kinetic.mat')
+    file = os.path.join('tests', 'payloads', 'tremor_kinetic.mat')
     data = read.read(file)
     actual = list(data.keys())
     assert actual == ['Flex', 'Ext', 'Angle', 'triangle', 'Keyboard']
 
 
 def test_read_with_channels_smoke_test():
-    file = os.path.join('.', 'payloads', 'tremor_kinetic.mat')
+    file = os.path.join('tests', 'payloads', 'tremor_kinetic.mat')
```
Comment (bbelderbos) [inline tests/test_read.py:17]: the basepath is repeated a lot, make it a constant

File: tests/test_read.py [hunk 2]
```diff
@@ -59,7 +60,7 @@ def test_parse_mat_events(data_setup):
 
 def test_parse_mat_keyboard_codes(data_setup):
     actual = read._parse_mat_keyboard(data_setup['mat_keyboard'])
-    assert actual == ['J', '9', '.', '5', 'S']
+    assert actual['codes'] == ['J', '9', '.', '5', 'S']
```
Comment (bbelderbos) [inline tests/test_read.py:63]: this failed for me

File: tests/test_read.py [hunk 3]
```diff
@@ -28,19 +28,20 @@ def test_exit_code_no_smr_file():
     assert pytest_wrapped_e.value.code == 1
 
 
-# def test_message_no_smr_file():
-#     read.read('file.smr')
-#     actual = capdfd.readouterr()[0].strip()
-#     expected = 'Processing .smr files is currently not supported.\n ' \
-#                'In Spike2 export the data to .mat and start over.'
-#     assert actual == expected
+def test_message_no_smr_file(capfd):
```
Comment (bbelderbos) [inline tests/test_read.py:31]: commented test fix, need to work around the `sys.exit`

File: tests/test_read.py [hunk 4]
```diff
@@ -3,18 +3,18 @@
 import numpy as np
 import scipy.io as sio
 
-import read
+from spike2py import read
```
Comment (bbelderbos) [inline tests/test_read.py:6]: needed to fix imports

**Review by bbelderbos — COMMENTED** [review]
Hey Marty, check out my test fixes.

File: tests/test_read.py (hunk 4 above)
Comment (MartinHeroux) [reply tests/test_read.py:6]: Makes sense.

File: tests/test_read.py (hunk 1 above)
Comment (MartinHeroux) [reply tests/test_read.py:17]: Will do.

Thanks.

File: tests/test_read.py (hunk 2 above)
Comment (MartinHeroux) [reply tests/test_read.py:63]: Thanks. I will look into it.

**Pull request merged**